### DISCUSSION TITLE
LND v0.18.4-beta Release Candidate

### DISCUSSION THREAD

**Discussion by saubyk** [lightningnetwork/lnd#9304] [General]
The release candidate for the upcoming minor LND release v0.18.4 has been published.

This release is primarily in support of the channel state machine changes required for processing and storing custom Taproot script leaves. This allows for implementation of custom channels which is required to support taproot assets over Lightning.

The release can be found at:
https://github.com/lightningnetwork/lnd/releases/tag/v0.18.4-beta.rc1

Detailed release are available at:
https://github.com/lightningnetwork/lnd/blob/0-18-4-branch/docs/release-notes/release-notes-0.18.4.md

**Release highlights:**
- Custom channel support for Taproot asset channels
- Compatibility with Bitcoin core version 28
- Fix for an unnecessary size expansion of the payment request
- Fixed a nil pointer issue in route fee estimation for blinded routes

Reports of issues found during the testing will be highly appreciated 🫂

Thanks

**Comment by ViktorT-11** [comment]
Hi @jvxis,

> However, I cannot change channel policies from my test node's side. Every time I try, I receive the message:
`channel from self node has no policy`

The issue might be related to your node not being fully synced with the graph yet, so let's start by solving that issue first to see if that resolves the problem.

1. Do your logs show any ongoing syncing activity? This is typically indicated by rapidly added log entries with details about various channels.  
2. If you run `lncli describegraph`, does the output include information about numerous channels? (For a properly synced node, it should display details for thousands of channels.)

   - **If no**: Try connecting to another large node using `lncli connect` and check if this initiates the syncing process for your node.  
   - **If yes**: Does the output contain the specific channel for which you're attempting to update the channel policy for?

**Reply by jvxis** [reply]
> Hi @jvxis,
> 
> > However, I cannot change channel policies from my test node's side. Every time I try, I receive the message:
> > `channel from self node has no policy`
> 
> The issue might be related to your node not being fully synced with the graph yet, so let's start by solving that issue first to see if that resolves the problem.
> 
> 1. Do your logs show any ongoing syncing activity? This is typically indicated by rapidly added log entries with details about various channels.
> 2. If you run `lncli describegraph`, does the output include information about numerous channels? (For a properly synced node, it should display details for thousands of channels.)
>    
>    * **If no**: Try connecting to another large node using `lncli connect` and check if this initiates the syncing process for your node.
>    * **If yes**: Does the output contain the specific channel for which you're attempting to update the channel policy for?

Thanks. I deleted the message because I installed another instance of the new release on another machine, and it worked. So this is not a problem with this version and may be an exclusive problem with my PI installation.

I checked `lncli describegraph` and yes, there are many nodes there... 
So I enabled `Clearnet` and connected to some nodes. Let's see if it gets synced now.

Thanks,
JVX
